src/hipscat/io/paths.py

```diff
@@ -87,7 +90,10 @@ def get_healpix_from_path(path: str) -> HealpixPixel:
 
 
 def pixel_catalog_files(
-    catalog_base_dir: FilePointer, pixels: List[HealpixPixel], storage_options: Dict | None = None
+    catalog_base_dir: FilePointer,
+    pixels: List[HealpixPixel],
+    storage_options: Dict | None = None,
```

Comment:
We've been putting `storage_options` as the last argument in the signature, just for convention.